src/polygon/src/polygon.cpp

```diff
@@ -123,7 +123,7 @@ void CheckGeometry(const std::vector<ivec3> &triangles,
 
 void Dump(const PolygonsIdx &polys) {
   for (auto poly : polys) {
-    std::cout << "polys.push_back({" << std::setprecision(9) << std::endl;
+    std::cout << "polys.push_back({" << std::setprecision(15) << std::endl;
```

Comment:
Seems like there's not much point dumping this format now - can we update it to the format you now use in the `txt` files?

Reply:
Sure. My major concern is that the new format can be a bit hard to read (as I want to keep the parser simple), and the old c++ format is more self-explaining.